### PR TITLE
Add Postmark ingress support to ActionMailbox

### PR DESCRIPTION
**Pull request by tomazy**
### Summary

This PR adds [Postmark](https://postmarkapp.com/) ingress support to ActionMailbox.

### CODE REVIEW

**Comment by georgeclaghorn** [comment]
Can you add the new route to the necessary tests [here](https://github.com/rails/rails/blob/9f1a07af0499080c9fd8815705a03a4c7e8fb506/railties/test/commands/routes_test.rb)?

File: railties/test/commands/routes_test.rb [hunk 1]
```diff
@@ -220,81 +222,86 @@ class Rails::Command::RoutesTest < ActiveSupport::TestCase
...
       URI               | /rails/action_mailbox/mailgun/inbound_emails/mime(.:format)
       Controller#Action | action_mailbox/ingresses/mailgun/inbound_emails#create
-      --[ Route 7 ]--------------
+      --[ Route 8 ]--------------
       Prefix            | rails_conductor_inbound_emails
       Verb              | GET
       URI               | /rails/conductor/action_mailbox/inbound_emails(.:format)
       Controller#Action | rails/conductor/action_mailbox/inbound_emails#index
-      --[ Route 8 ]--------------
-      Prefix            |
```
Comment (georgeclaghorn) [inline railties/test/commands/routes_test.rb]: I know it’s annoying, but the trailing space on this line and the other Prefix lines below needs to be preserved.

**Comment by georgeclaghorn** [comment]
Closer! It looks like there are a few more tests that need updating.

**Comment by georgeclaghorn** [comment]
Thank you!

**Comment by tomazy** [comment]
Thanks for such a quick merge :)

**Pull request merged**